faust_large_message_serializer/clients/retrieving_client.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         config: LargeMessageSerializerConfig
     ):
         self._config = config
+        self._client = config.get_blob_storage_client()
```

Comment:
Now just make it the parameter ;-)

Reply:
Or is the config used anywhere else in the class?

Reply:
Ideally, the config has a method createStorer and createRetriever

Reply:
no, xD and I don't get why pycharm doesn't tell me about it :/